pynapple/io/neurosuite.py

```diff
@@ -371,7 +371,43 @@ def load_lfp(self, filename=None, channel=None, extension='.eeg', frequency=1250
                 time_units = 's',
                 time_support = time_support,
                 columns=channel)
-
+            
+    def read_neuroscope_intervals(self, name=None, path2file=None):
+        """
+        This function reads .evt files in which odd raws indicate the beginning 
+        of the time series and the even raws are the ends. If the file is not present
+        in the nwb, it loads the events from the nwb directory.
+        
+    
+        Parameters
+        ----------
+        name: str
+            name of the .evt file. The name of the file in the loading directory should
+            be:   "Session + name + .evt", example: A0001.rem.evt
+            For using this function you just need the name between the two dots.
+        
+        path2file: str
+            Path of the file you want to load.
+    
+        Returns
+        -------
+        IntervalSet
+            Containing two columns corresponding to the start and end of the intervals.
+    
+        """
+        isets = self.load_nwb_intervals(name)
+        if str(type(isets)) !="<class 'pynapple.core.interval_set.IntervalSet'>":
```

Comment:
I would remove this. if the previous condition is good no need to go through this.